packages/shared/styles/components/organisms/SfTable.scss

```diff
@@ -12,59 +12,46 @@
   }
   &__heading {
     align-items: center;
-    padding: var(--table-heading-padding, var(--spacer-sm) 0);
-    background-color: var(--table-heading-background, transparent);
-    @include border(--table-border-heading, 0 0 2px 0, solid, var(--c-primary));
+    padding: var(--spacer-sm);
+    background-color: transparent;
+    border: solid var(--c-primary);
+    border-width: 0 0 2px 0;
   }
   &__header {
-    @include font(
-      --table-header-font,
-      var(--font-weight--normal),
-      var(--font-size--base),
-      1.4,
-      var(--font-family--secondary)
-    );
-    color: var(--table-header-color, var(--c-text));
+    font-weight: var(--font-weight--normal);
+    font-size: var(--font-size--base);
+    font-family: var(--font-family--secondary);
+    line-height: 1.4;
+    color: var(--c-text);
   }
   &__row {
-    padding: var(--table-row-padding, var(--spacer-xs) 0);
-    background-color: var(--table-row-background);
+    padding: var(--spacer-sm);
+    border: solid var(--c-light);
+    border-width: 0 0 2px 0;
     box-shadow: var(--table-row-box-shadow);
-    @include border(--table-row-border, 0 0 2px 0, solid, var(--c-light));
   }
   &__data {
-    color: var(--table-data-color, var(--c-dark-variant));
-    @include font(
-      --table-data-font,
-      var(--font-weight--normal),
-      var(--font-size--base),
-      1.6,
-      var(--font-family--secondary)
-    );
+    color: var(--c-dark-variant);
+    font-weight: var(--font-weight--normal);
+    font-size: var(--font-size--base);
+    font-family: var(--font-family--secondary);
+    line-height: 1.6;
   }
   &__data,
   &__header {
     box-sizing: border-box;
-    flex: var(--table-column-flex, 0 0 calc(100% / var(--_table-column-width)));
-    padding: var(--table-column-padding, 0);
-    text-align: var(--table-column-text-align, left);
-    &:nth-child(even) {
-      order: var(--_table-column-order, 1);
-    }
+    flex: var(--table-column-flex, 0 0 calc(100% / var(--table-column-width)));
+    padding: 0;
+    text-align: left;
   }
   &--no-border {
-    --table-row-border-width: 0;
+    .sf-table__row {
+      border-width: 0;
+    }
   }
   @include for-desktop {
-    --table-row-padding: var(--spacer-sm) var(--spacer-sm);
-    --table-heading-padding: var(--spacer-sm) var(--spacer-sm);
-    --table-heading-background: transparent;
     --table-column-flex: 1;
```

Comment:
why you left this one + 2 others?

Reply:
this is for the column no? I'm just not so sure we should remove it. @filrak